packages/agoric-cli/src/commands/inter.js

```diff
@@ -266,33 +388,68 @@ For example:
 
   bidCmd
     .command('cancel')
-    .description('Print a request to exit a bid offer')
+    .description('Exit a bid offer')
```

Comment:
we have no "exit" operation, only "try exit", which may fail

Reply:
I wonder how I check whether it failed.

Reply:
Oh, that's a good one. `tryExit()` is async and returns nothing. You can call `E(seat).hasExited()`, and it will promptly return a boolean as to whether it has finished exiting. You can use `E(seat).getExitSubscriber()` to be notified when it exits. But there's nothing that tells you whether `tryExit()` succeeded.

Reply:
And for it to be useful from a CLI (or any other off-chain client), it has to be visible via chainStorage/RPC.

Reply:
> tryExit() is async and returns nothing. 

I'm wrong again.

It does return a promise for the result of the attempt to exit.